### DISCUSSION TITLE
how to disable password with helm chart (v3.10.0 and appVersion 4.14.0)

### DISCUSSION THREAD

**Discussion by yoongeon1206** [coder/code-server#6403] [Q&A]
I have installed code-server with helm chart v3.10.0 and appVersion 4.14.0 in my kubernetes environment
I try to disable password with the option --auth none. but it doesn't work. is there anyone who know how to resolve this issue? 

1. I added the option "--auth none" to extraArgs in value.yaml. and then I deployed code-server with helm chart.
value.yaml
extraArgs:
   - --auth none 
   
however I got the error below.
" error Unknown option --auth none"

2. and I try again as below in value.yaml

value.yaml
extraArgs:
   - "--auth none" 
  
I got the same error message. Could you tell me how to disable password in helm chart? 
" error Unknown option --auth none"

**Comment by code-asher** [comment]
Check out the comments for extraArgs:
https://github.com/coder/code-server/blob/eb8099ff89eaca72cc9051f440d817cfd0e24084/ci/helm-chart/values.yaml#L58-L69

Basically, each argument needs to be on its own line, so:

extraArgs:
  - --auth
  - none
